### PR TITLE
Fix swallowed errors in upup subdirectory.

### PR DESCRIPTION
**Pull request by alrs**
I found some unhandled err variables in packages under the `upup` directory.

### CODE REVIEW

**Comment by alrs** [comment]
I've signed the CLA.

File: upup/pkg/fi/cloudup/vspheretasks/attachiso.go [hunk 1]
```diff
@@ -92,11 +92,18 @@ func (_ *AttachISO) CheckChanges(a, e, changes *AttachISO) error {
 // RenderVSphere executes the actual task logic, for vSphere cloud.
 func (_ *AttachISO) RenderVSphere(t *vsphere.VSphereAPITarget, a, e, changes *AttachISO) error {
 	startupScript, err := changes.BootstrapScript.ResourceNodeUp(changes.IG)
+	if err != nil {
+		return err
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/vspheretasks/attachiso.go]: should we `fmt.Errorf`?

File: upup/pkg/fi/cloudup/vspheretasks/attachiso.go [hunk 2]
```diff
@@ -92,11 +92,18 @@ func (_ *AttachISO) CheckChanges(a, e, changes *AttachISO) error {
...
+	}
 	startupStr, err := startupScript.AsString()
 	if err != nil {
 		return fmt.Errorf("error rendering startup script: %v", err)
 	}
 	dir, err := ioutil.TempDir("", *changes.VM.Name)
+	if err != nil {
+		return err
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/vspheretasks/attachiso.go]: should we `fmt.Errorf`?

**Comment by alrs** [comment]
Thanks, @chrislovecnm, I've expanded the returned err variables to fmt.Errorf() with some extended explanations.

**Pull request merged**